manifests/puppetmaster.pp

```diff
@@ -25,9 +25,7 @@
     default:  { $json_package = 'rubygem-json' }
   }
 
-  package { $json_package:
-    ensure  => installed,
-  }
+  ensure_packages($json_package)
```

Comment:
The [documentation](https://github.com/puppetlabs/puppetlabs-stdlib#ensure_packages) says it should be a list of packages.